package.json

```diff
@@ -58,7 +58,7 @@
   },
   "peerDependencies": {
     "react": "^15.4.2",
-    "react-helmet": "^3.3.2",
+    "react-helmet": ">3.3.2",
```

Comment:
its a little more disruptive since it forces consumers to change the version of react-helmet they supply. otherwise their build will break and likely cause a bit of confusion.